src/main/scala/com/fullcontact/interview/RecordFinder.scala

```diff
@@ -1,7 +1,59 @@
+/*
+Author: FullContact
+Modified By: Jeff Willingham
+Creation Date: unknown
+Modification Date: 2020-07-28
+Purpose: This script inputs Queries.txt and Records.txt, performs transformations and writes outputs per specifications outlined in README.md.
+*/
+
 package com.fullcontact.interview
 
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+import org.apache.spark.sql.functions._
+
 object RecordFinder {
   def main(args: Array[String]): Unit = {
-    // TODO: Implement your job here
+    val spark = SparkSession // instantiate spark session
+      .builder()
+      .appName("RecordFinder")
+      .master("local[*]")
+      .getOrCreate
+    val queriesDF: DataFrame = spark // read in Queries.txt as a DataFrame
+      .read
+      .format("text")
+      .load("Queries.txt")
+      .withColumnRenamed("value", "queryID")
+    val recordsDF: DataFrame = spark // read in Records.txt as a DataFrame, and create two fields
+      .read
+      .format("text")
+      .load("Records.txt")
+      .withColumn("idArray", split(col("value"), "\\s"))
+      .withColumn("idArrayExploded", explode(col("idArray")))
+    queriesDF // join inputs to create and write Output1.txt
+      .join(recordsDF, col("queryID") === col("idArrayExploded"), "inner")
+      .withColumn("output1", concat_ws(":", col("queryID"), col("value")))
+      .select("output1")
```

Comment:
Is there a difference between writing this as
```scala
      .select(concat_ws(":", col("queryID"), col("value")) as "output1")
```
and what you have currently? If so, what?